sdcm/cluster_k8s/mini_k8s.py

```diff
@@ -189,6 +188,9 @@ def start_k8s_software(self) -> None:
         """)
         self.host_node.remoter.run(f"sudo -E bash -cxe '{script}'")
 
+    def stop_k8s_software(self):
+        self.host_node.remoter.run('kind delete cluster', ignore_status=True)
```

Comment:
Shouldn't we make sure that cluster is deleted? If it fails to be deleted then it will keep computer resources used and we won't be able to create new cluster.

Reply:
kind can fail on cluster deletion by multiple reasons, not all of them means that cluster wasn't actually deleted.
It is safe to fail later on cluster creation